widget/radio_item.go

```diff
@@ -0,0 +1,165 @@
+package widget
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/internal/widget"
+	"fyne.io/fyne/theme"
+)
+
+var _ fyne.Widget = (*RadioItem)(nil)
+var _ desktop.Hoverable = (*RadioItem)(nil)
+var _ fyne.Tappable = (*RadioItem)(nil)
+
+func newRadioItem(label string, onTap func(*RadioItem)) *RadioItem {
+	i := &RadioItem{Label: label, onTap: onTap}
+	i.ExtendBaseWidget(i)
+	return i
+}
+
+// RadioItem is a single radio item to be used by RadioGroup.
+type RadioItem struct {
```

Comment:
I was thinking more about this, I can't think of any benefit to exporting this type?

Reply:
Hm … me neither 😄. Will unexport it.